packages/api-elements/lib/define-value-of.js

```diff
@@ -93,10 +93,8 @@ module.exports = (namespace) => {
     return undefined;
   }
 
-  // TODO e instanceof EnumElement fails because of dependency injection
-  // checking for e.element === 'enum' as a temporary  walkaround
-  const isEnumElement = e => (e.element === 'enum' || e instanceof EnumElement);
-  const isPlural = e => (e instanceof ArrayElement) || (e instanceof ObjectElement);
```

Comment:
ObjectElement is an instanceof ArrayElement and thus this additional check was redundant.